src/main/java/kg/jarkyn/cobspecserver/Request.java

```diff
@@ -0,0 +1,4 @@
+package kg.jarkyn.cobspecserver;
+
+public class Request {
```

Comment:
one of the collaborators is expecting it as a return value, hence this placeholder
i find this happens with top-down approach, especially when you already have a fairly good idea of where you are going